pages/resources.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Head from "pages/head";
+import { Box, Container, Link } from "@material-ui/core";
+
+function Resources() {
+  return (
+    <div>
+      <Head />
+      <Container>
+        <Box maxWidth={680} mx="auto">
+          <h2>Resources</h2>
+          <p>
+            Below are a list of resources we have made available to the public.
+          </p>
+
+          {/* Segment talking about our data */}
+
+            <h3>Our Data</h3>
+              To get a copy of our latest dataset, follow{" "}
+            <Link href="https://github.com/glp-volunteers/pb-dashboard/blob/master/db/police_killings.csv">
+              <b> this link.</b>
+            </Link>
+              To read about how we gather our data, please visit our{" "}
+            <Link href="/methodology">
```

Comment:
Since this is an internal link, using the `Link` component that NextJs offers will be more performant. 

```suggestion
            <Link component={NextLink} href="/methodology">
```

Note: You'll need to import the component like you did in the Header component with:

```
import { Link as NextLink } from "next/link";
```